src/networking.c

```diff
@@ -539,6 +539,116 @@ get_mac_address(const gchar *const ifname)
 	return macaddr;
 }
 
+/*
+ * Get and store the iptable rules set in the
+ * network namespace. These need to be re-applied
+ * within the container.
+ *
+ * \param config \ref cc_oci_config
+ *
+ * \return \c true on success, else \c false.
+ */
+static gboolean
+get_iptable_rules(struct cc_oci_config *config)
+{
+	gboolean ret;
+	GError *error = NULL;
+	gint exit_status = 0;
+	char *cmd = "iptables-save";
+	char *args[] = { cmd, NULL };
+
+	if (! config) {
+		return false;
+	}
+
+	g_debug("Running command : %s", cmd);
+	ret = g_spawn_sync (NULL,
+              args,
+              NULL,
+              G_SPAWN_SEARCH_PATH | G_SPAWN_STDERR_TO_DEV_NULL,
```

Comment:
Different distros may have this under different path.
eg Ubuntu has this under /sbin/iptables, while Clear has this under /usr/bin.
Docker and hyper codebases do not provide the full path as well.

Also, since G_SPAWN_SEARCH_PATH is provided, the program will not be run from the current directory.